components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -129,29 +130,73 @@ + (NSDictionary *)catalogMetadata {
   };
 }
 
+#pragma mark - UIViewController
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  self.contentView.frame = self.view.bounds;
```

Comment:
Since all of your views appear to be using autoresize masks there shouldn't need to be manual frame-setting here. 

Please turn of autoresizing masks (`.translatesAutoresizingMaskIntoConstraints = NO;`) or switch to a full constraint-based layout. Mixing the two is likely to cause trouble at some point plus it's difficult to reason about.

Reply:
I agree. I will do it in a separate PR to clean it up. https://github.com/material-components/material-components-ios/issues/6842 was created to track the work here.